cmd/stop.go

```diff
@@ -26,6 +26,8 @@ var stopCmd = &cobra.Command{
 	Short: "Stop a task",
 	Long:  `Stops working on a task and updates the end time.`,
 	Run: func(cmd *cobra.Command, args []string) {
```

Comment:
This question is a little late. But where does this function get the command and strings from?

Reply:
What do you mean?